app/src/main/java/com/quarkworks/apartmentgroceries/MyApplication.java

```diff
@@ -10,14 +13,43 @@
     private static final String TAG = MyApplication.class.getSimpleName();
 
     private static Context context;
+    private String sessionToken;
+    private String userId;
+    private String groupId;
 
     @Override
     public void onCreate() {
         super.onCreate();
         context = this;
+        RealmConfiguration config = new RealmConfiguration.Builder(context).deleteRealmIfMigrationNeeded().build();
+        Realm.setDefaultConfiguration(config);
     }
 
     public static Context getContext() {
         return context;
     }
+
+    public String getSessionToken() {
+        return sessionToken;
+    }
+
+    public void setSessionToken(String sessionToken) {
+        this.sessionToken = sessionToken;
+    }
+
+    public String getUserId() {
+        return userId;
+    }
```

Comment:
Instead of holding on to references, to these variables, you should save and retrieve them from SharedPreferences.